http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/RuntimeTemplateValues.scala

```diff
@@ -0,0 +1,202 @@
+                                          proxyServerName: String,
+                                          jupyterUserScriptUri: String,
+                                          jupyterUserScriptOutputUri: String,
+                                          jupyterStartUserScriptUri: String,
+                                          jupyterStartUserScriptOutputBaseUri: String,
+                                          jupyterServiceAccountCredentials: String,
+                                          loginHint: String,
+                                          jupyterServerExtensions: String,
+                                          jupyterNbExtensions: String,
+                                          jupyterCombinedExtensions: String,
+                                          jupyterLabExtensions: String,
+                                          jupyterNotebookConfigUri: String,
+                                          jupyterNotebookFrontendConfigUri: String,
+                                          googleClientId: String,
+                                          welderEnabled: String,
+                                          notebooksDir: String,
+                                          customEnvVarsConfigUri: String,
+                                          memLimit: String) {
+
+  def toMap: Map[String, String] =
+    this.getClass.getDeclaredFields.map(_.getName).zip(this.productIterator.to).toMap.mapValues(_.toString)
+
+}
+
+case class RuntimeTemplateValuesConfig(runtimeProjectAndName: RuntimeProjectAndName,
+                                       stagingBucketName: Option[GcsBucketName],
+                                       runtimeImages: Set[RuntimeImage],
+                                       initBucketName: Option[GcsBucketName],
+                                       jupyterUserScriptUri: Option[UserScriptPath],
+                                       jupyterStartUserScriptUri: Option[UserScriptPath],
+                                       serviceAccountKey: Option[ServiceAccountKey],
+                                       userJupyterExtensionConfig: Option[UserJupyterExtensionConfig],
+                                       defaultClientId: Option[String],
+                                       welderEnabled: Boolean,
+                                       auditInfo: AuditInfo,
+                                       imageConfig: ImageConfig,
+                                       welderConfig: WelderConfig,
+                                       proxyConfig: ProxyConfig,
+                                       clusterFilesConfig: ClusterFilesConfig,
+                                       clusterResourcesConfig: ClusterResourcesConfig,
+                                       clusterResourceConstraints: Option[RuntimeResourceConstraints])
+object RuntimeTemplateValuesConfig {
+  def fromCreateCluster(createCluster: CreateCluster,
```

Comment:
I think the name is accurate because it's from a `CreateCluster` object, which is not the API request but the payload of the create-cluster PubSub message. 

I really want to rename `CreateCluster` and the other models in `leoPubsubMessageSubscriberModels` but I'm hesitant to do it in this PR -- maybe I will in my next PR.

Reply:
Sounds good. Class names that are verbs like `CreateCluster` throw me off  🤷‍♂